apps/src/applab/DesignToggleRow.jsx

```diff
@@ -39,7 +40,11 @@ module.exports = React.createClass({
   },
 
   handleScreenChange: function (evt) {
-    this.props.onScreenChange(evt.target.value);
+    var screenId = evt.target.value;
+    if (screenId === 'New screen...') {
```

Comment:
might be worth extracting a variable for this string, since you're assuming it's exactly the same in two places